.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Checkout code
```

Comment:
Please do not move LICENSE file. It belongs to the repository, not particular library. (I can't put comment on LICENSE file)